docs/sql-ref-syntax-aux-cache-uncache-table.md

```diff
@@ -19,4 +19,21 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Drop all cached entries associated with the table from the RDD cache
```

Comment:
Is it safe to say this just undoes a `CACHE TABLE`? or if I create a DataFrame from the table, and cache it, would it uncache that too? I actually don't know, just making sure the 'all cached entries' means what we intend. Whatever the answer it might bear a little more explanation.